README.md

```diff
@@ -66,8 +66,8 @@ npm start
 
 ## Search
 
-We are using Algolia [docsearch](https://www.algolia.com/docsearch) for our
-documentation search. The algolia index for Kong is maintained by Algolia through their
+We use Algolia [docsearch](https://www.algolia.com/docsearch) for our
```

Comment:
To fix issue I mention at l79, maybe instead "Documentation for our open-sources projects uses Algolia for search. The Algolia index for the OSS docs is maintained ..."